python/lsst/ts/m2com/mock/mock_control_open_loop.py

```diff
@@ -0,0 +1,445 @@
+            File path of the static transfer matrix.
+        skiprows : `int`, optional
+            Number of lines to skip (int) at the start of the file. (the
+            default is 7)
+        """
+
+        dataframe = pd.read_csv(filepath, skiprows=skiprows)
+        data = np.array(dataframe.iloc[:, 0])
+
+        self._static_transfer_matrix = data.reshape(NUM_ACTUATOR, NUM_ACTUATOR)
+
+    def update_actuator_steps(self, actuator_steps):
+        """Update the actuator steps.
+
+        Parameters
+        ----------
+        actuator_steps : `numpy.ndarray [int]`
+            Actuator steps.
+
+        Raises
+        ------
+        `ValueError`
+            When the length of actuators does not match.
+        `ValueError`
+            When the data type is not integer.
+        """
+
+        if len(actuator_steps) != NUM_ACTUATOR:
+            raise ValueError("The length of actuators does not match.")
+
+        if actuator_steps.dtype != int:
+            raise ValueError("The data type is not integer.")
+
+        self.actuator_steps = actuator_steps
+
+    def correct_inclinometer_angle(self, angle, offset=0.94):
+        """Calibrate the inclinometer's value and make sure to limit the
+        resulting value to the indicated range: (-270, 90).
+
+        Note: This function is translated from vendor's original LabVIEW code.
+        The hard-coded range is related to the configuration of inclinometer
+        in the mirror assembly.
+
+        Parameters
+        ----------
+        angle : `float`
+            Inclinometer angle in degree.
+        offset : `float`, optional
+            Offset in degree. The default value is hard-coded in the LabVIEW's
+            project. (the default is 0.94)
+
+        Returns
+        -------
+        angle_correct : `float`
+            Corrected inclinometer angle in degree.
+        """
+
+        angle_offset = angle + offset
+        origin = 360 if (0 <= angle_offset < 90) else 0
+
+        angle_correct = 180 - angle_offset - origin
+
+        # Make sure the calculated angle value is limited to this range
+        if angle_correct > 90:
+            angle_correct = 90
+        elif angle_correct < -270:
+            angle_correct = -270
+
+        return angle_correct
+
+    def get_forces_mirror_weight(self, angle):
+        """Get the forces that bear the weight of mirror.
+
+        Note: This function is translated from vendor's original LabVIEW code.
+        This is just an estimation of forces.
+
+        Parameters
+        ----------
+        angle : `float`
+            Inclinometer angle in degree.
+
+        Returns
+        -------
+        forces : `numpy.ndarray`
+            Forces to support the mirror in Newton.
+        """
+
+        forces = np.zeros(NUM_ACTUATOR)
+
+        angle_correct = self.correct_inclinometer_angle(angle)
+
+        num_axial_actuators = NUM_ACTUATOR - NUM_TANGENT_LINK
+        force_mirror_weight = MIRROR_WEIGHT_KG * 9.8
```

Comment:
Maybe make the gravitation acceleration a constant instead of using the value directly here? If you rather use the value, maybe add a comment?

Reply:
Updated in the commit: [b244b0a](https://github.com/lsst-ts/ts_m2com/pull/3/commits/b244b0aad8f4c2a0d09a08aeac6524d5ec86b72e). Thanks!